src/Raven.Server/Documents/Replication/ChangeVectorParser.cs

```diff
@@ -7,7 +7,9 @@ namespace Raven.Server.Documents.Replication
     public static class ChangeVectorParser
     {
         public const string RaftTag = "RAFT";
+        public const string TrxnTag = "TRXN";
```

Comment:
What is actually `Trxn`? Transaction compare exchange?

Reply:
Just an way to indicate that, yes. We can call it something else, couldn't think of anything more meaningful in 4 chars.

Reply:
Then maybe `TXCX`? TX - from "transaction", CX - from compare exchange

Reply:
I don't like that, I chose `TRXN` to look like transaction.
I would rather have something meaningful, rather than abbervation

Reply:
ok